src/components/main/TreeModal.js

```diff
@@ -24,26 +28,36 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-const { useItem, useItems, useOwnItems, useChildren } = hooks;
+const { useItem, useItems, useOwnItems, useChildren, useSharedItems } = hooks;
 
 const TreeModal = ({ itemIds, open, title, onClose, onConfirm, prevent }) => {
   const { t } = useTranslation();
   const classes = useStyles();
-  const { data: ownItems, isLoading } = useOwnItems();
+  const { data: ownItems, isLoading: isOwnItemsLoading } = useOwnItems();
+  // bug: get only shared items with write/admin rights
+  const { data: sharedItems, isLoading: isSharedItemsLoading } =
+    useSharedItems();
   const [selectedId, setSelectedId] = useState(null);
   const { data: items, isItemLoading } = useItems(itemIds);
 
-  const buildExpandedItems = () => {
+  const buildExpandedItems = (rootId) => {
     if (items && !items.isEmpty()) {
       // suppose all items are in the same parent
       const parentIds = getParentsIdsFromPath(items.first().path) || [];
-      const newExpandedItems = [ROOT_ID, ...parentIds];
-      return newExpandedItems;
+      // return expanded list depending current root id
+      // define root depending on whether is the root parent is in the owned items
+      const rootItemId = parentIds[0];
+      const oItem = ownItems.find(({ id }) => id === rootItemId);
```

Comment:
Nitpick: convert to `Boolean()` and use a more explicit name, e.g. `isRootItemOwned` ?